spec/models/provider_relationship_spec.rb

```diff
@@ -3,6 +3,13 @@
 require "rails_helper"
 
 RSpec.describe ProviderRelationship, type: :model do
+  let(:lead_provider) { create(:lead_provider) }
+  let(:delivery_partner) { create(:delivery_partner) }
+  let(:cohort) { create(:cohort) }
+  let!(:provider_relationship) do
+    ProviderRelationship.create(lead_provider: lead_provider, delivery_partner: delivery_partner, cohort: cohort)
+  end
```

Comment:
oh oh, what does the `let!` do that `let` does not?

Reply:
It works like `before`. Runs even if you don't call it